internal/server/migrations.go

```diff
@@ -111,9 +111,10 @@ func (a *API) addResponseRewrites() {
 		return newResponse.Items
 	})
 	addResponseRewrite(a, http.MethodGet, "/v1/grants/:id", "0.12.2", migrateUserGrantToIdentity)
-	addResponseRewrite(a, http.MethodGet, "/v1/grants", "0.12.2", func(newResponse *api.ListResponse[api.Grant]) []identityGrant {
-		return api.NewListResponse(newResponse.Items, migrateUserGrantToIdentity).Items
-	})
+	// addResponseRewrite(a, http.MethodGet, "/v1/grants", "0.12.2", func(newResponse *api.ListResponse[api.Grant]) []identityGrant {
+	// 	return api.NewListResponse(newResponse.Items, migrateUserGrantToIdentity).Items
+	// })
+	//TODO: Fix List Migrations (need pagination or to search through all)
```

Comment:
Setting pagination to nil is what I would expect where pagination wasn't in this previous version. Might need a response re-write also that removes the pagination struct.